scripts/take_all_genomes.py

```diff
@@ -87,8 +87,8 @@ def read_params():
 bins_dictionary = {}
 
 for i in range(len(cluster_tsv)):
-	contig = str(cluster_tsv["contig"][i]).split(" ")[0]
-	sample = str(cluster_tsv["contig"][i]).split(args.binsplit_char)[0]
+	contig = cluster_tsv["contig"][i].split(" ")[0]
+	sample = cluster_tsv["contig"][i].split("_NODE")[0]
```

Comment:
I suggest we update `args.binsplit_char` to default to `_NODE` rather than hardcoding here. And we should change the corresponding call within `mags_generation.smk` and yaml generation in `binning_preparation.smk`.

Reply:
Testing it now